docs/docs-beta/sidebars.ts

```diff
@@ -1,495 +1,441 @@
-        {
-          type: 'category',
-          label: 'Authentication & access control',
-          items: [
-            {
-              type: 'category',
-              label: 'Role-based Access Control',
-              link: {
-                type: 'doc',
-                id: 'dagster-plus/access/rbac',
-              },
-              items: [
+            type: 'category',
+            label: 'Build',
+            items: [
                 {
-                  type: 'autogenerated',
-                  dirName: 'dagster-plus/access/rbac',
+                    type: 'category',
+                    label: 'Create a pipeline',
+                    items: [
+                        'guides/data-assets',
+                        'guides/metadata',
+                        'guides/partitioning',
+                        'guides/external-assets',
+                    ],
                 },
-              ],
-            },
-            {
-              type: 'category',
-              label: 'Single Sign-on (SSO)',
-              items: [
-                'dagster-plus/access/authentication/azure-ad-sso',
-                'dagster-plus/access/authentication/google-workspace-sso',
-                'dagster-plus/access/authentication/okta-sso',
-                'dagster-plus/access/authentication/onelogin-sso',
-                'dagster-plus/access/authentication/pingone-sso',
-              ],
-            },
-            {
-              type: 'category',
-              label: 'SCIM provisioning',
-              items: [
                 {
-                  type: 'link',
-                  label: 'Azure Active Directory',
-                  href: 'https://learn.microsoft.com/en-us/azure/active-directory/saas-apps/dagster-cloud-provisioning-tutorial',
+                    type: 'category',
+                    label: 'Configure behavior',
+                    items: [
+                        'guides/configuring-assets',
+                        'todo',
+                    ],
                 },
                 {
-                  type: 'doc',
-                  label: 'Okta',
-                  id: 'dagster-plus/access/authentication/okta-scim',
+                    type: 'category',
+                    label: 'Abstracting logic',
+                    items: [
+                        'guides/asset-factories',
+                        'guides/asset-factories-with-deps',
+                    ],
                 },
-              ],
-            },
-          ],
+                {
+                    type: 'category',
+                    label: 'Common use cases',
+                    items: [
+                        'guides/ingesting-data',
+                        'guides/transform-dbt',
+                        'todo',
+                    ],
+                },
+            ],
         },
         {
-          type: 'doc',
-          id: 'dagster-plus/saved-views',
+            type: 'category',
+            label: 'Scale',
```

Comment:
Big fan of this direction.

My initial thought is that "Scale" is unclear, as first think of scaling systems / compute.

I would expect this section, along with some of the above to be "Develop"

Reply:
What would you see as the difference between build and develop? With Scale I was trying to convey scale out into production, e.g. you build an asset, but to make it resilient, you add asset checks.

Reply:
I would opt for them to be consolidated, as they both fit into "building", but I can see what you were going for. I would probably go for something like the following. With `>` indicated expandable:

```
Getting started

 - Overview
 - Installation
 - Quickstart
 - Tutorial

Build

 - Overview
 - Create pipelines >
 - Configuration >
 - Automation >
 - Abstraction >
     - Factory patterns
     - DSLs
 - Testing >
     - Asset checks
     - Data freshness checks
     - Unit testing

Deploy

 - Overview
 - Self host >
 - Dagster+ >
 - Code locations
 - Environment variables and secrets

Use cases

 - <Common use cases moved here>

Understand

 - ...

About

 - ...
```

Reply:
I was writing my own comment about Build vs Develop when I saw this - I agree with Colton. Awhile back I put out an RFC for a new sidebar that used both of these terms, and the feedback was overwhelmingly that it was redundant and kinda confusing.

We may be able to stick with Scale (or something similar) if we turn off auto-collapse for these categories (but keep them short), so users could see the category contents and gain context without clicking. I see Colton's point about some items fitting better under Build, but my concern is then that it would become a pretty beefy section.

Reply:
I'll go with Scale for now to get this merged, we can always change the heading name painlessly. Maybe once we have enough items under build vs. scale it'll become more apparent what to name or where to consolidate